tests/test_pandas_dataset.py

```diff
@@ -12,6 +12,27 @@
 
 class TestPandasDataset(unittest.TestCase):
 
+    def run_encapsulated_test(self, expectation_name, filename):
```

Comment:
Is this a legacy test runner sneaking back in? Can't all of these be run using the new test runner?

Reply:
Sorry, I should've put a note on the PR before I went to bed last night.

This is a legacy test runner, which needs to be replaced before this PR is done. The tests need to be rewritten in the process, and I ran out of time for it yesterday.